index.html

```diff
@@ -52,7 +52,12 @@ <h2>
       <h3>
         Adding a model to a document
       </h3>
-      <aside class="issue" data-number="19"></aside>
+      <aside class="example">
+         <p>The following example ads a binary glTF model to a document.</p>
+         <pre>
+           &lt;model&gt; &lt;source src="assets/example.glb" type="model/gltf-binary"&gt; &lt;model&gt;
+         </pre>
+      </aside>
```

Comment:
Let's start with the most simple inclusion... leaving it up to the user agent to do content negotiation to figure out the type. 

```suggestion
         <p>The following example shows the simplest way to add a 3D model to a document:</p>
         <pre class="html">
           &lt;model src="3d-assets/car"&lt;model&gt;
         </pre>
         <p>
           By using content negotiation the user agent relies on the server to send back a 3D model in a 
           format the user agent supports.
         </p>
```

We can then build up from there to introduce `<source>` for explicit type declaration.